client/src/components/Link/Link.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components/macro";
+import React from "react";
+
+const Button = styled.button`
+  background-color: #fff;
+  border-radius: 50px;
+  border: none;
+  font-family: "Orbitron", Arial, sans-serif;
```

Comment:
You could left that behind, because you already declared it for the hole document

```suggestion
```

Reply:
👍🏻